runtimes/talos/src/main/java/com/talosvfx/talos/runtime/vfx/assets/BaseAssetProvider.java

```
@@ -46,4 +47,9 @@ public <T> T findAsset (String assetName, Class<T> clazz) {
 		return getAssetHandler(clazz).findAsset(assetName);
 	}
 
+	@Override
+	public <T> GameAsset<T> findGameAsset (String assetName, Class<T> clazz) {
+		return null;
```

Prefer to throw an exception here or make it an abstract method